### PR TITLE
Addition of author (NMJ) to manuscript

### PR DESCRIPTION
**Pull request by nafisajadavji**
Details of addition:     
    github: nafisajadavji
    name: Nafisa M. Jadavji
    initials: NMJ
    orcid: 0000-0002-3557-7307
    twitter: nafisajadavji
    email: njadav@midwestern.edu
    contributions:
      - Writing - Original Draft
      - Writing - Review & Editing
    affiliations:
      - Biomedical Science, Midwestern University, Glendale, Pheonix, United States of America
      -Department of Neuroscience, Carleton of University, Ottawa, Ontario, Canada
    coi:
      string: "None"
      lastapproved: 
    funders: American Heart Association (20AIREA35050015)

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -267,3 +267,20 @@ authors:
       string: "None"
       lastapproved: !!str 2020-04-07
     funders: the Gordon and Betty Moore Foundation (GBMF 4552)
+
```
Comment (rando2) [inline content/metadata.yaml:270]:
```suggestion
  -
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -267,3 +267,20 @@ authors:
...
+    initials: NMJ
+    orcid: 0000-0002-3557-7307
+    twitter: nafisajadavji
+    email: njadav@midwestern.edu
+    contributions:
+      - Writing - Original Draft
+      - Writing - Review & Editing
+    affiliations:
+      - Biomedical Science, Midwestern University, Glendale, Pheonix, United States of America
+      - Department of Neuroscience, Carleton of University, Ottawa, Ontario, Canada
```
Comment (rando2) [inline content/metadata.yaml:282]:
```suggestion
      - Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
```
Just checking!

File: content/metadata.yaml [hunk 3]
```diff
@@ -267,3 +267,20 @@ authors:
...
+    coi:
+      string: "None"
+      lastapproved:
```
Comment (agitter) [inline content/metadata.yaml:285]:
```suggestion
      lastapproved: !!str 2020-04-09
```

This line indicates the date you last approved your conflict of interest statement.

File: content/metadata.yaml (hunk 2 above)
Comment (nafisajadavji) [reply content/metadata.yaml:282]: No it wasn't. Thanks for catching that!

File: content/metadata.yaml [hunk 4]
```diff
@@ -267,3 +267,20 @@ authors:
...
+    name: Nafisa M. Jadavji
+    initials: NMJ
+    orcid: 0000-0002-3557-7307
+    twitter: nafisajadavji
+    email: njadav@midwestern.edu
+    contributions:
+      - Writing - Original Draft
+      - Writing - Review & Editing
+    affiliations:
+      - Biomedical Science, Midwestern University, Glendale, Pheonix, United States of America
```
Comment (cbrueffer) [inline content/metadata.yaml:303]: Typo

```suggestion
      - Biomedical Science, Midwestern University, Glendale, Phoenix, United States of America
```

File: content/metadata.yaml [hunk 5]
```diff
@@ -293,3 +293,16 @@ authors:
...
       string: "None"
       lastapproved: !!str 2020-04-08
+  -
+    github: nafisajadavji
+    name: Nafisa M. Jadavji
+    initials: NMJ
+    orcid: 0000-0002-3557-7307
+    twitter: nafisajadavji
+    email: njadav@midwestern.edu
+      - Biomedical Science, Midwestern University, Glendale, Phoenix, United States of America
```
Comment (rando2) [inline content/metadata.yaml:303]:
```suggestion
affiliations:
    - Biomedical Science, Midwestern University, Glendale, AZ, United States of America
```

File: content/metadata.yaml [hunk 6]
```diff
@@ -293,3 +293,16 @@ authors:
...
+      - Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
```
Comment (rando2) [inline content/metadata.yaml:304]:
```suggestion
    - Department of Neuroscience, Carleton University, Ottawa, Ontario, Canada
```

**Pull request merged**